youtube_dl/extractor/pietsmiet.py

```diff
@@ -0,0 +1,99 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_urllib_parse_unquote,
+)
+from ..utils import (
+    unescapeHTML,
+    js_to_json,
+    int_or_none,
+)
+
+
+class PietsmietIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?pietsmiet\.de/gallery/(categories|playlists)/[\w-]+/(?P<id>\d+)-.*/?'
+    _TESTS = [
+        {
+            'url': 'https://www.pietsmiet.de/gallery/categories/8-frag-pietsmiet/29844-fps-912',
+            'info_dict': {
+                'id': '29844',
+                'ext': 'mp4',
+                'title': 'Was würdet ihr die Maus fragen? 🎮 Frag PietSmiet #912',
+            },
+            'params': {
+                'skip_download': True,  # m3u8 downloads
+            }
+        }, {
+            'url': 'https://www.pietsmiet.de/gallery/playlists/646-metal-gear-solid-1/19804-metal-gear-solid-1-sniper-wolf-rematch',
+            'info_dict': {
+                'id': '19804',
+                'ext': 'mp4',
+                'title': 'SNIPER WOLF REMATCH 🎮 Metal Gear Solid #9'
+            },
+            'params': {
+                'skip_download': True,  # m3u8 downloads
+            }
+        }
+    ]
+
+    def _real_extract(self, url):
+        page_id = self._match_id(url)
+        webpage = self._download_webpage(url, page_id)
+        data_video_config = self._search_regex(
+            r'var config =(.*?)\};\n', webpage, 'video config', flags=re.DOTALL)
+        data_video_config = data_video_config.replace(']', '],', 1) + '}'
```

Comment:
This should be fixed in `js_to_json`.

Reply:
I looked into this but I'm unsure on how to do that. I don't want to fix code of this importance to the project without guidance from a project maintainer.

The current error is `ERROR: 41807: Failed to parse JSON  (caused by JSONDecodeError("Expecting ',' delimiter: line 7 column 6 (char 377)",));`

And the corresponding JSON is here (weird formatting left in tact on purpose): https://gist.github.com/mhartkorn/f081c5a38692a19444b46a2c51d3a4bb